src/main/java/com/example/demo/service/UserService.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.service;
+
+import com.example.demo.domain.User;
+import com.example.demo.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository repository;
+
+    public void save(User user){
+        repository.save(user);
```

Comment:
/wtd use User dto that not contains the id and create a user mapper  to map the user dto to the user entity then save it